### PR TITLE
bug(#3895):fixed the button alignment and ui as per the design

### LINKED ISSUE #3895: 🐛 Bug Report: Not align with style button in new project modal
The button in the create new project modal is not aligning with the design spec by @alexbass86 

# The wanted design
New Project

# The existing design
https://user-images.githubusercontent.com/61761153/190987201-c32826c8-e59b-4f64-857f-720b09f7bb88.mov

### PR DESCRIPTION
**Pull request by KlausMikhaelson**
Close #3895 

## PR Details
Fixed the alignment and Ui as per the design

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by abrl91** [comment]
run branch

**Review by abrl91 — APPROVED** [review]

**Comment by abrl91** [comment]
@KlausMikhaelson 
image

**Comment by KlausMikhaelson** [comment]
awesome thank you ! btw @Asmit2952 it doesn't matter but was it a part of hacktoberfest as well ? if yes than can you add the label ?

**Comment by abrl91** [comment]
> awesome thank you ! btw @Asmit2952 it doesn't matter but was it a part of hacktoberfest as well ? if yes than can you add the label ?

@souravjain540 ?

**Comment by souravjain540** [comment]
I have read the guidelines of hacktoberfest thoroughly and I can say that I am 100% sure, if you have an approved pull request in a hacktoberfest participating repository which amplication/amplication is, it will get counted

so don't worry about tags and labels :) @KlausMikhaelson

**Comment by KlausMikhaelson** [comment]
yeah just saw that it has already been updated on my hacktoberfest profile my bad, thank you !

File: packages/amplication-client/src/Project/NewProject.scss [hunk 1]
```diff
@@ -8,3 +8,7 @@ $image-height: 130px;
     height: $image-height;
   }
 }
+
+.new-project__instructions {
+  color: #A3A8B8;
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Project/NewProject.scss:13]: @abrl91 this should be a variable

File: packages/amplication-client/src/Project/NewProject.tsx [hunk 2]
```diff
@@ -110,11 +110,13 @@ const NewProject = ({ onProjectCreated }: Props) => {
                 autoComplete="off"
               />
               <Button
+                className={CLASS_NAME}
+                style={{backgroundColor:"#7950ED", color:"#FFFFFF"}}
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Project/NewProject.tsx:114]: we should not use inline style

**Review by yuval-hazaz — COMMENTED** [review]
@abrl91 please see my comments and open an issue so the community can fix it

**Comment by abrl91** [comment]
@KlausMikhaelson can you open a pull request to fix it?
In your fork, you need to checkout to branch `next` and pull the changes from the remote. Then checkout to a new branch

**Comment by KlausMikhaelson** [comment]
yeah @abrl91 got it !

**Pull request merged**